boards/xtensa/esp32/esp32-devkitc/configs/knsh/defconfig

```diff
@@ -62,4 +62,5 @@ CONFIG_START_YEAR=2011
 CONFIG_SYSTEM_NSH=y
 CONFIG_TESTING_GETPRIME=y
 CONFIG_TESTING_OSTEST=y
+CONFIG_TESTING_OSTEST_FPUTESTDISABLE=y
```

Comment:
@gustavonihei should we change fpu  testing instead:
https://github.com/apache/nuttx-apps/blob/master/testing/ostest/Kconfig#L81
```
if ARCH_FPU && SCHED_WAITPID && BUILD_FLAT
```
and the related Makefile/Source code change.

Reply:
Nice suggestion, I think it makes sense to automatically disable when not under Flat Build.